platform/app/cypress/integration/measurement-tracking/OHIFCornerstoneToolbar.spec.js

```diff
@@ -406,20 +406,42 @@ describe('OHIF Cornerstone Toolbar', () => {
     cy.get('@moreBtn').click();
     cy.get('.tooltip-toolbar-overlay').should('not.exist');
   });
+*/
+  it('check if Flip tool will flip the image in the viewport', () => {
+    cy.get('@viewportInfoMidLeft').should('contains.text', 'R');
+    cy.get('@viewportInfoMidTop').should('contains.text', 'A');
 
-  it('check if Flip V tool will flip the image vertically in the viewport', () => {
     //Click on More button
-    cy.get('@moreBtn').click();
-    //Verify if overlay is displayed
-    cy.get('.tooltip-toolbar-overlay').should('be.visible');
+    cy.get('@moreBtnSecondary').click();
 
-    //Click on Flip V button
-    cy.get('[data-cy="flip v"]').click();
-    cy.get('@viewportInfoMidLeft').should('contains.text', 'R');
-    cy.get('@viewportInfoMidTop').should('contains.text', 'F');
+    //Click on Flip button
+    cy.get('[data-cy="flip-horizontal"]').click();
+    cy.waitDicomImage();
+    cy.get('@viewportInfoMidLeft').should('contains.text', 'L');
+    cy.get('@viewportInfoMidTop').should('contains.text', 'A');
+  });
 
-    //Click on More button to close it
-    cy.get('@moreBtn').click();
-    cy.get('.tooltip-toolbar-overlay').should('not.exist');
-  });*/
+  it('checks if stack sync is preserved on new display set and uses FOR', () => {
```

Comment:
Here is the primary test for this issue.

Reply:
Thanks a lot